11.3.Методы_списков._Часть_1/11.3.12.py

```diff
@@ -0,0 +1,10 @@
+n = int(input())
+b = []
+
+for i in range(n):
```

Comment:
Только что похвалил что называешь верно переменные xD
```suggestion
for _ in range(n):
```